tests/sles4sap/monitoring_services.pm

```diff
@@ -70,7 +70,7 @@ sub configure_hanadb_exporter {
     # Add monitoring resource in the HA stack
     wait_for_idle_cluster;
     if (get_var('HA_CLUSTER') and is_node(1)) {
-        my $hanadb_msl = "msl_SAPHana_$args{rsc_id}";
+        my $hanadb_msl = get_var('USE_SAP_HANA_SR_ANGI') ? "mst_SAPHanaCtl_$args{rsc_id}" : "msl_SAPHana_$args{rsc_id}";
```

Comment:
Similar code is here too: https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/361cabc57a69faf9d13627854f8b66f50b2a8f30#diff-1a0a1f54a03bede88230cb0854402e50c18e0ef4ef77757d2f5cf170c228c70bR82

Maybe, as we have this dependency, we can create a library function that get the name and use it in both place

This function is also a good place to document where the name come from --> https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19902/files#diff-7a35318b3b4c3de3f8a63ea45ac54d6cba9b83b68913c960152f2309814026d8R39